src/aind_metadata_mapper/gather_metadata.py

```diff
@@ -271,14 +278,19 @@ def get_subject(self) -> Optional[dict]:
             contents = self._get_file_from_user_defined_directory(file_name=file_name)
         return contents
 
-    def get_procedures(self) -> Optional[dict]:
-        """Get procedures metadata"""
+    def get_procedures(self, subject_id: str) -> Optional[dict]:
```

Comment:
I just added a new `get_procedures` method here:
https://github.com/AllenNeuralDynamics/aind-metadata-mapper/blob/f539a0400d9dad2812bcb6c4dc5b79302e2f833a/src/aind_metadata_mapper/utils.py#L47 Could you use that instead rather than defining a new one here? Seems like the one I made works somewhat differently but the end result should be the same, right?